content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1182,7 +1182,7 @@ allowed by *either* the RBAC or ABAC policies is allowed.
 
 When the kube-apiserver is run with a log level of 5 or higher for the RBAC component
 (`--vmodule=rbac*=5` or `--v=5`), you can see RBAC denials in the API server log
-(prefixed with `RBAC DENY:`).
+(prefixed with `RBAC:`).
```

Comment:
```suggestion
(prefixed with `RBAC`).
```

Reply:
Hi @liggitt, the prefixes on the rest of the page (specifically [this](https://github.com/kubernetes/website/blob/1b47d0191b61c77aef77101116a431c975c6d450/content/en/docs/reference/access-authn-authz/rbac.md#referring-to-subjects) subsection) preserve their colons, which is why I did the same for this instance. Wouldn't it help maintain uniformity across the page?

Reply:
Dropping the colon would make the stated prefix match both the old and the message

Reply:
Understood, thanks for the clarification.